chernoAlieu.Habit-Tracker/Habit-Tracker/Program.cs

```diff
@@ -0,0 +1,240 @@
+
+		while (!Int32.TryParse(numberInput, out _) || Convert.ToInt32(numberInput) < 0)
+		{
+			Console.WriteLine("\n\nInvalid number. Try again");
+			numberInput = Console.ReadLine();
+		}
+
+		int finalInput = Convert.ToInt32(numberInput);
+
+		return finalInput;
+	}
+
+	private static void GetAllRecords()
+	{
+		Console.Clear();
+		using (var connection = new SqliteConnection(connectionString))
+		{
+			connection.Open();
+			var tableCmd = connection.CreateCommand();
+			tableCmd.CommandText =
+				$"SELECT * FROM drinking_water ";
+
+			List<DrinkingWater> tableData = new();
+			SqliteDataReader reader = tableCmd.ExecuteReader();
+
+			if (reader.HasRows)
+			{
+				while (reader.Read())
+				{
+					tableData.Add(
+						new DrinkingWater
+						{
+							Id = reader.GetInt32(0),
+							Date = DateTime.ParseExact(reader.GetString(1), "dd-MM-yy", new CultureInfo("en-US")),
+							Quantity = reader.GetInt32(2)
+						});
+				}
+			}
+			else
+			{
+				Console.WriteLine("No rows found");
+			}
+
+			connection.Close();
+			Console.WriteLine("----------------------------------------------\n");
+			foreach (var drinkingWater in tableData)
+			{
+				Console.WriteLine($"{drinkingWater.Id} - {drinkingWater.Date.ToString("dd-MM-yyyy")} - Quantity: {drinkingWater.Quantity}");
+			}
+			Console.WriteLine("----------------------------------------------\n");
+		}
+	}
+
+	private static void Delete()
+	{
+		Console.Clear();
+		GetAllRecords();
+
+		var recordId = GetNumberInput("\n\nPlease take the Id of the record you" +
+		                              " want to delete or type 0 to go back to main menu");
+
+		using (var connection = new SqliteConnection(connectionString))
+		{
+			connection.Open();
+			var tableCmd = connection.CreateCommand();
+			tableCmd.CommandText = $"DELETE from drinking_water WHERE Id = '{recordId}'";
+
+			int rowCount = tableCmd.ExecuteNonQuery();
+
+			if (rowCount == 0)
+			{
+				Console.WriteLine($"\n\nRecord with Id {recordId} doesn't exist. \n\n");
+				Delete();
+			}
+		}
+
+		Console.WriteLine($"\n\nRecord with Id {recordId} was deleted.\n\n");
+		GetUserInput();
+	}
+
+	private static void Update()
+	{
+		Console.Clear();
+		GetAllRecords();
+
+		var recordId = GetNumberInput("\n\nPlease type the Id of the record you would like to update." +
+		                              "Type 0 to go back to the main menu");
+		using (var connection = new SqliteConnection(connectionString))
+		{
+			connection.Open();
+
+			var checkCmd = connection.CreateCommand();
+			checkCmd.CommandText = $"SELECT EXISTS(SELECT 1 FROM drinking_water WHERE Id = {recordId})";
+			int checkQuery = Convert.ToInt32(checkCmd.ExecuteScalar());
+
+			if (checkQuery == 0)
+			{
+				Console.WriteLine($"\n\nRecord with Id {recordId} doesn't exist.\n\n");
+				connection.Close();
+				Update();
+			}
+
+			string date = GetDateInput();
+			int quantity = GetNumberInput("\n\nPlease insert number of glasses or other measure of your " +
```

Comment:
You can use @ to print your strings verbatim. Check this out: 
![image](https://user-images.githubusercontent.com/108711476/216686316-cc4bb809-d0df-4d0e-9364-1a4915bd8008.png)